File: src/C#/task_1/pattern.cs
using System;

class Program
{
    // Функция для печати алфавитной пирамиды в обратном порядке
    static void PrintAlphabetPyramid(int N)
    {
        // Итерация от N до 1 для печати строк сверху вниз
        for (int row = N; row > 0; row--)
        {
            // Итерация от N до row для печати символов в строке справа налево
            for (int letter = N; letter >= row; letter--)
            {
                // Печать символа, начиная с соответствующего номеру строки
                Console.Write(Convert.ToChar('A' + letter - 1) + " ");
            }
            // Переход на новую строку
            Console.WriteLine();
        }
    }

    static void Main(string[] args)
    {
        Console.WriteLine("Введите N:");
        int N = Convert.ToInt32(Console.ReadLine());
        PrintAlphabetPyramid(N);
    }
}

File: src/Kotlin/task_1/pattern.kt
fun printAlphabetPyramid(N: Int) {
    // Итерация от N до 1 для печати строк сверху вниз
    for (row in N downTo 1) {
        // Итерация от N до row для печати символов в строке справа налево
        for (letter in N downTo row) {
            // Печать символа, начиная с соответствующего номеру строки
            print((letter + 64).toChar() + " ")
        }
        // Переход на новую строку
        println()
    }
}

fun main() {
    val N = readLine()!!.toInt()
    printAlphabetPyramid(N)
}
